win64svc/src/common.h

```diff
@@ -1,6 +1,3 @@
-//
-// Created by Vlad Ilyushchenko on 06/08/2016.
```

Comment:
i quite liked that comment :)

Reply:
Don't worry, git history will never forget ;)

Reply:
thank you Alex! It is great to see you here!